.github/workflows/pr_notification.yml

```
@@ -0,0 +1,140 @@
+
+env:
+  USERNAME: "돌아왔다 PR 리뷰 시간"
+  AVATAR_URL: "https://i.ibb.co/GvfVG729/R1280x0-fjpg.jpg"
+
+jobs:
+  notify:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)